### PR TITLE
Rearranges the antagonist panel, cleans it up code-wise and appearance-wise

### PR DESCRIPTION
**Pull request by ghost**
(The devil and ninja options don't appear twice in the final version, don't worry.)

The antagonist panel has always struck me as unpolished and very cramped in regards to how information is displayed. Additionally, the antagonist ordering made no sense; revolutionaries, for instance, were at the top, and traitors about halfway down.

I changed this in a few ways:
* Added padding between each word and its |, so instead of "traitor|LOYAL" you see "traitor | LOYAL"
* Rearranged antagonists in a more sensible way, with common ones like traitors and changelings at the top and cults and the like on the bottom
* Certain antags (i.e. devils) will no longer show up for mobs that can never have that antagonist
* Standardized grammar across the board
* Replaced most instances of "loyal" with "mindshielded" or "not mindshielded"
* Slightly expanded the panel to prevent text wrap

With regards to code, I moved around things to make more sense. Additionally, I put a lot of antagonists under eligibility checks (i.e. `ishuman(current)`, `issilicon(current)`) to prevent them from appearing on mobs that can't ever possess the antagonist. I also changed the mindshield implant text to display dynamically instead of having a completely separate text for those with implants, that had a button to convert them to an antagonist - which didn't do anything, since they were implanted!

### CODE REVIEW

File: code/datums/mind.dm [hunk 1]
```diff
@@ -759,21 +766,21 @@
...
+	usr << browse(out, "window=edit_memory[src];size=575x600")
 
 
 /datum/mind/Topic(href, href_list)
 	if(!check_rights(R_ADMIN))
 		return
 
 	if (href_list["role_edit"])
-		var/new_role = input("Select new role", "Assigned role", assigned_role) as null|anything in get_all_jobs()
+		var/new_role = input("Select new role", "Assigned role", assigned_role) as null | anything in get_all_jobs()
```
Comment (RemieRichards) [inline code/datums/mind.dm]: Well it's obvious how you did this

**Comment by ChangelingRain** [comment]
I'd put monkey, devil, and ninja below abductor.
also is it possible to make this the black HTML UI because that would be nice.

**Comment by Xhuis** [comment]
I'd like to put those antagonists at the bottom but it'd be wonky.
Black HTML UI should be easy enough, I'll see how it looks.

**Comment by Xhuis** [comment]
# Yes, I know about devil and ninja appearing twice! It's fixed.

**Comment by kevinz000** [comment]
tfw i'll miss all admin panels being old fashioned near-plaintext white panels instead of uis.
;-;

**Pull request merged**